lib/ProofSet.js

```diff
@@ -146,12 +146,14 @@ module.exports = class ProofSet {
 
       // verify proofs
       const results = await _verify({
-        document, suites, proofSet, purpose, documentLoader, expansionMap
+        document, suites, proofSet, purpose, documentLoader, expansionMap,
```

Comment:
```suggestion
        document, suites, proofSet, purpose, documentLoader, expansionMap
```

Remove hanging comma.

Reply:
Removed.